Automattic/wp-calypso
# Post-checkout: Quick Start upsell copy fix

#### Changes proposed in this Pull Request

* Fixing grammar issues raised in https://github.com/Automattic/wp-calypso/issues/36028

#### Testing instructions

- Purchase a plan without a domain, you should see the Quick Start upsell screen.
- Make sure the first paragraph includes the word "**than**": "[...] in first time *than* you ever though [...]"


Fixes https://github.com/Automattic/wp-calypso/issues/36028
